mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,60 @@
+import CommunitySlider, {
+    type SliderProps,
+} from '@react-native-community/slider';
+import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    thumbTitleValue: string;
+    containerStyle?: StyleProp<ViewStyle>;
+    value: number;
+    thumbTitleValueWidth?: number;
+} & Omit<SliderProps, 'style' | 'value'>;
+
+const defaultMinSliderValue = 0;
+const defaultMaxSliderValue = 120;
+const defaultValueWidth = 70;
```

Comment:
What is default value?

Reply:
Value is required porps or it bad practice?

Reply:
Ok, let's make them more usual, for example from 0 to 10 and 0 as default

Reply:
fix